pydantic/json_schema.py

```diff
@@ -803,7 +803,9 @@ def list_schema(self, schema: core_schema.ListSchema) -> JsonSchemaValue:
         self.update_with_validations(json_schema, schema, self.ValidationsMapping.array)
         return json_schema
 
-    def tuple_positional_schema(self, schema: core_schema.TuplePositionalSchema) -> JsonSchemaValue:
+    @deprecated('tuple_positional_schema is deprecated. Use tuple_schema instead.')
+    @final
+    def tuple_positional_schema(self, schema: core_schema.TupleSchema) -> JsonSchemaValue:
```

Comment:
I think perhaps we want to use a `deprecated` pattern similar to that enforced in this recent PR: #8294.

Specifically, here's an example: https://github.com/pydantic/pydantic/pull/8294/files#diff-2d54a873f0e89e361f635a68aff651e9507fe82cce1881943cafba93295315ae

@Viicos, thanks for the guidance on proper usage of `deprecated` going forward 🚀 😄.